APE0.rst

```diff
@@ -157,8 +157,8 @@ In the case of a vacancy partway through a term, a by-election will be held.
 The term of the newly-elected member runs for the remainder of the term of
 the member they are replacing. 
 
-For the initial election of Coordination Committee members, two seats will have
-terms of one year, one has a term of two years, and one has a term of three
+For the initial election of Coordination Committee members (four at the time instead of five), two seats will have
+terms of one year, one has a term of two years (two here now that there are five), and one has a term of three
```

Comment:
Probably can be worded better...

Reply:
I don't see a good reason for mentioning the terms for a 4-member CoCo at all, it just makes the text more confusing.

Reply:
Removing a line from established APE might be controversial?

Feel free to suggest alternate wordings. Thanks!

Reply:
I applied your suggestion. I am not a lawyer either. 😸